src/components/NewPosition/DepositSelector/DepositSelector.tsx

```diff
@@ -138,7 +138,7 @@ export const DepositSelector: React.FC<IDepositSelector> = ({
       <Grid container className={classes.sectionWrapper} style={{ marginBottom: 8 }}>
         <Grid container className={classes.selects} direction='row' justifyContent='space-between'>
           <Grid className={classes.selectWrapper}>
-            <Typography className={classes.inputLabel}>Pair token A</Typography>
+            {/* <Typography className={classes.inputLabel}>Pair token A</Typography> */}
```

Comment:
remove commented code it there will be no need for it later

Reply:
updated